src/modules/cookies/cookies-context.tsx

```diff
@@ -44,7 +44,7 @@ export function AppCookiesProvider({
   const darkmode = useCookie<boolean>(
     DARKMODE_COOKIE_NAME,
     initialCookies.darkmode ?? undefined,
-    (i) => i == true,
+    (i) => (typeof i === 'string' ? i === 'true' : i == true),
```

Comment:
Could we do a triple equals in the last part now after checking types?

Reply:
And will `i` always be a string?

Reply:
Not entirely sure about all of this tbh, but it can be `string | boolean | null | undefined`. I've updated to use triple equals.